target_mvd/urls.py

```diff
@@ -16,8 +16,13 @@
 from django.contrib import admin
 from django.urls import include, path
 
+from users.views import FacebookLogin
+
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('dj-rest-auth/', include('dj_rest_auth.urls')),
     path('dj-rest-auth/registration/', include('dj_rest_auth.registration.urls')),
+    path('dj-rest-auth/facebook/', FacebookLogin.as_view(), name='fb_login'),
+    path('accounts/', include('allauth.urls')),
+    path('users/', include('users.urls')),
```

Comment:
Maybe declare `name` for all the routes in order to use `reverse` in test files and other places without the necessity of change in each of them in case of modifying one of the routes